pkg/revision/image/image.go

```diff
@@ -0,0 +1,256 @@
+package image
```

Comment:
pkg/revision-controller/image/image.go

Reply:
This package includes the revision and revision controller, the image and git are revision, so revision is more suitable.

Reply:
Like Knative revision and azure container apps revisions https://learn.microsoft.com/en-us/azure/container-apps/revisions, Revision is used to specify a version of Function Serving. It's not related to how the function is built(from git or from source in image) or where the source code is from.

This repo is actually used to ***update*** revision of function serving based on git source changes or image changes. So the git and image stuff is not part of revisions, they're triggers or controllers of revisions.

Reply:
Done